department-of-veterans-affairs/vets-website
# MHV-58425 MHV-58352 Preface Conditions Details Title With Health Conditions

## Summary

The ConditionsList page has been updated with the "SCT" code for the condition names. Additionally, the ConditionDetail page now includes "Health conditions:" before the condition names.

## Related issue(s)
[MHV-58425](https://jira.devops.va.gov/browse/MHV-58425) - Preface Conditions Details title with "Health Conditions:
[MHV-58352](https://jira.devops.va.gov/browse/MHV-58352) - Do not parse out the "SCT" code in Conditions Details title

MHV-58425:
From [Amelia Hacking](https://jira.devops.va.gov/secure/ViewProfile.jspa?name=amelia.hacking%40va.gov) : 
We do want the title of the details view to say "Health conditions: {}insert condition title {}" for consistency across domains.
 
Slack discussion for context: https://dsva.slack.com/archives/C04ER7MHX5E/p1716326848508209?thread_ts=1716322426.630759&cid=C04ER7MHX5E

User Story: As a Medical Records User, I want to   * , so that I can    

GIVEN

WHEN

THEN

Feature Flag Y/N ? N

DataDog/Analytics Y/N ? N

Manual Testing Y/N ?  Y

Automated Testing Y/N ? N

Accessibility Testing Y/N ? Y

UCD Validation Y/N Y

Acceptance Criteria:

AC1 The4 title of the details view will say "Health conditions: 

AC2 Verify with UCD-Amelia

AC3
AC4
AC5

Links to UCD:

Figma Link:

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

MHV-58352:
Currently, in Conditions Details, we parse the  H1 title to remove any parenthetical codes that may appear. Remove any such parsing.

AC1: The title of the condition in details matches the title in the list

AC2: The title is not parsed in any way, instead displaying exactly what comes from the JSON

User Story: As a Medical Records User, I want to see content without any extra characters so that I am not confused by content.

Feature Flag Y/N ? N

DataDog/Analytics Y/N ? N

Manual Testing Y/N ? Y

Automated Testing Y/N ? N

Accessibility Testing Y/N ? N

UCD Validation Y/N N

Acceptance Criteria:

AC1: The title of the condition in details matches the title in the list

AC2: The title is not parsed in any way, instead displaying exactly what comes from the JSON

AC3
AC4
AC5

Links to UCD:

Figma Link:

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

## Testing done
Conducted UI testing to ensure the expected behavior. I updated existing unit test for new components.

## Screenshots
![Screenshot 2024-05-28 at 12 03 29 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/d55c7ee8-92cd-4ce7-b795-5f18c21eb238)
![Screenshot 2024-05-28 at 12 03 40 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/dbb8f24c-c853-4865-825b-c9e5b75a2166)


## What areas of the site does it impact?
The modification affects  ConditionDetail and ConditionList of the Medical Records pages.


## Acceptance criteria
1. ConditionDetail page "Health Conditions:" before condition name.
2. ConditionList has had the "SCT" code added back in.

### Quality Assurance & Testing


### Error Handling


## Acceptance criteria


### Quality Assurance & Testing


### Authentication
